src/components/DataTable/DataTable.jsx

```diff
@@ -0,0 +1,260 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import { findAllChildComponents } from '../../util/child-component';
+
+import Checkbox from '../Checkbox/Checkbox';
+import ScrollTable from '../ScrollTable/ScrollTable';
+
+const {
+	Thead,
+	Tbody,
+	Tr,
+	Th,
+	Td
+} = ScrollTable;
+
+const boundClassNames = lucidClassNames.bind('&-DataTable');
+
+const {
+	any,
+	func,
+	object,
+	string,
+	bool,
+	arrayOf,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["table"], "madeFrom": ["Table","ScrollTable"]}
```

Comment:
Yeah, I hear you. I really wanted to keep `madeFrom` based strictly on composition so we could draw a dependency graph of our library. It might be overkill, but we could have a `links` sections that gives people the most useful links to other components.